README.md

```diff
@@ -1,98 +1,107 @@
-# AWSSNS &mdash; Amazon Simple Notification Service Library
+# AWSSNS 2.0.0 #
 
 This library implements [Amazon Simple Notification Service (SNS)](https://aws.amazon.com/documentation/sns/) actions in agent code.
 
-**To add this library to your application, add the following lines to the top of your agent code:**
+**To include this library in your project, add the following lines at the top of your agent code:**
 
-```
+```squirrel
 #require "AWSRequestV4.class.nut:1.0.2"
-#require "AWSSNS.agent.lib.nut:1.0.0"
+#require "AWSSNS.agent.lib.nut:2.0.0"
 ```
 
-**Note** [AWSRequestV4](https://github.com/electricimp/AWSRequestV4/) must be included **before** the AWSSNS library.
+**Important** The [AWSRequestV4](https://github.com/electricimp/AWSRequestV4/) library must be included **before** the AWSSNS library.
 
 ![Build Status](https://cse-ci.electricimp.com/app/rest/builds/buildType:(id:Awssns_BuildAndTest)/statusIcon)
 
-## Class Usage
+## Class Usage ##
 
-### constructor(*region, accessKeyID, secretAccessKey*)
+### constructor(*region, accessKeyID, secretAccessKey*) ###
 
-The constructor takes the following parameters, all of which are required:
+#### Parameters ####
 
-Parameter              | Type           | Description
----------------------- | -------------- | -----------
-*region*                 | string         | AWS region (eg. `"us-west-2"`)
-*accessKeyId*            | string         | IAM Access Key ID
-*secretAccessKey*        | string         | IAM Secret Access Key
+| Parameter | Type | Required? | Description |
+| --- | --- | --- | --- |
+| *region*  | String | Yes | An AWS region (eg. `us-west-2`) |
+| *accessKeyId* | String | Yes | Your IAM Access Key ID |
+| *secretAccessKey* | String | Yes | Your IAM Secret Access Key |
 
-#### Example
+#### Example ####
 
 ```squirrel
+#require "AWSRequestV4.class.nut:1.0.2"
+#require "AWSSNS.agent.lib.nut:2.0.0"
+
 const AWS_SNS_ACCESS_KEY_ID     = "<YOUR_ACCESS_KEY_ID>";
 const AWS_SNS_SECRET_ACCESS_KEY = "<YOUR_SECRET_ACCESS_KEY_ID>";
 const AWS_SNS_URL               = "<YOUR_SNS_URL>";
```

Comment:
AWS_SNS_URL is not used. All examples use AWS_SNS_TOPIC_ARN. Please verify that this the same, and if so updated constant to `const AWS_SNS_TOPIC_ARN = "YOUR_TOPIC_ARN_HERE";`